docs/structured-streaming-programming-guide.md

```diff
@@ -2202,7 +2202,7 @@ streamingDatasetOfString.writeStream.foreach(
 In Java, you have to extend the class `ForeachWriter` ([docs](api/java/org/apache/spark/sql/ForeachWriter.html)).
 {% highlight java %}
 streamingDatasetOfString.writeStream().foreach(
-  new ForeachWriter[String] {
+  new ForeachWriter<String>() {
```

Comment:
This seems already fixed at https://github.com/apache/spark/commit/9a2cec9b1ee3ba095461d9c07c9932107228d42e . Can you remove this in your PR?